e_os.h

```diff
@@ -28,6 +28,29 @@
  * default, we will try to read at least one of these files
  */
 #  define DEVRANDOM "/dev/urandom", "/dev/random", "/dev/hwrng", "/dev/srandom"
+#  ifdef __linux
+#   define DEVRANDOM_WAIT   "/dev/random"
+/*
+ * Linux kernels 4.8 and later changes how their random device works and there
+ * is no reliable way to tell that /dev/urandom has been seeded -- getentropy(2)
+ * should be used instead.
+ */
+#   define DEVRANDOM_SAFE_KERNEL        4, 8
+/*
+ * Some operating systems do not permit select(2) on their random devices,
+ * defining this to zero will force the used of read(2) to extra one byte
+ * from /dev/random.
+ */
+#   define DEVRANDM_WAIT_USE_SELECT     1
```

Comment:
you probably want to add
`#ifndef DEVRANDM_WAIT_USE_SELECT`

Reply:
Personally, I prefer the system I used here which compile checks all code paths but I'll switch.

Reply:
no wait. I just wanted you to add a ifdef around this line.

Reply:
so I suggest
```C
#   ifndef DEVRANDM_WAIT_USE_SELECT
#    define DEVRANDM_WAIT_USE_SELECT    1
#   endif
```

Reply:
Yeah, my misunderstanding.